frontend/src/lib/components/apps/components/display/table/AppAggridTable.svelte

```diff
@@ -303,6 +303,11 @@
 							$componentControl[id] = {
 								agGrid: { api: e.api, columnApi: e.columnApi },
 								setSelectedIndex: (index) => {
+									if(index === -1) {
```

Comment:
should accept null or index rather than -1, unless that's an AgGrid convention

Reply:
I've updated this to be a null instead of -1